layers/+lang/extempore/packages.el

```diff
@@ -0,0 +1,97 @@
+;;; packages.el --- extempore layer packages file for Spacemacs.
+;;
+;; Copyright (c) 2020 Ben Swift
+;;
+;; Author: Ben Swift <ben@benswift.me>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+;;; Commentary:
```

Comment:
No commentary header in package files, documentation shall live in the readme instead.